app/controllers/anime_controller.rb

```diff
@@ -95,13 +111,9 @@ def filter_old
       @years = @filter_years
     end
 
-    if params[:sort]
-      @sort = params[:sort]
-    else
-      @sort = "all"
-    end
+    @sort = params[:sort] ||= 'all'
```

Comment:
Since you don't need to assign params[:sort] you can just do `params[:sort] || 'all'`